portal-web/test/functional/com/liferay/portalweb/macros/json/site/JSONSiteTemplateAPI.macro

```diff
@@ -0,0 +1,127 @@
+definition {
+
+	macro _addSiteTemplateByName {
+		Variables.assertDefined(parameterList = "${siteTemplateName}");
+
+		var portalURL = JSONCompany.getPortalURL();
+
+		var curl = '''
+			${portalURL}/api/jsonws/layoutsetprototype/add-layout-set-prototype \
+				-u test@liferay.com:test \
+				-d name=${name} \
+				-d description=${description} \
+				-d active=${active} \
+				-d layoutsUpdateable=${layoutsUpdateable} \
+				-d readyForPropagation=${readyForPropagation}
+		''';
+
+		com.liferay.poshi.runner.util.JSONCurlUtil.post("${curl}");
+	}
+
+	macro _applyTemplateToSite {
+		Variables.assertDefined(parameterList = "${groupId},${privateLayout},${templateUuid}");
+
+		var portalURL = JSONCompany.getPortalURL();
+
+		var curl = '''
+			${portalURL}/api/jsonws/layoutset/update-layout-set-prototype-link-enabled \
+                           -u test@liferay.com:test \
+                           -d groupId=${groupId}  \
+                           -d privateLayout=${privateLayout} \
+                           -d layoutSetPrototypeLinkEnabled=true \
+                           -d layoutSetPrototypeUuid=${templateUuid}
```

Comment:
layoutSetPrototypeUuid is expected as incoming parameter name

Reply:
Nice catch, thanks! I found actually a few unchanged parameter name so it was a good point! Resolved